src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -1242,21 +1244,21 @@ void QgsRasterLayerProperties::apply()
 
 void QgsRasterLayerProperties::updateSourceStaticTime()
 {
+  QgsProviderMetadata *metadata = QgsProviderRegistry::instance()->providerMetadata(
+                                    mRasterLayer->providerType() );
+  QVariantMap uri = metadata->decodeUri( mRasterLayer->dataProvider()->dataSourceUri() );
+  uri["allowTemporalUpdates"] = mWmstGroup->isChecked() ? QStringLiteral( "true" ) :  QStringLiteral( "false" );
```

Comment:
I'm not 100% sure but I think you could have used `bool` values directly here, give it a try.